primer-api/src/Primer/API.hs

```diff
@@ -1334,11 +1342,11 @@ getSelectionTypeOrKind = curry $ logAPI (noError GetTypeOrKind) $ \(sid, sel0) -
       def <- snd <$> findASTTypeDef allTypeDefs sel.def
       case sel.node of
         -- type def itself selected - return its kind
-        Nothing -> pure $ Kind $ viewTreeKind $ typeDefKind $ TypeDef.TypeDefAST def
+        Nothing -> pure $ Kind $ viewTreeKind' $ mkIdsK $ typeDefKind $ forgetTypeDefMetadata $ TypeDef.TypeDefAST def
         -- param node selected - return its kind
         Just (TypeDefParamNodeSelection p) ->
-          maybe (throw' $ ParamNotFound p) (pure . Kind . viewTreeKind . snd) $
-            find ((== p) . fst) (astTypeDefParameters def)
+          maybe (throw' $ ParamNotFound p.param) (pure . Kind . viewTreeKind . snd) $
+            find ((== p.param) . fst) (astTypeDefParameters def)
```

Comment:
I'm not a fan of the current behaviour. I think it would be better to (as you suggest) just say "kind".

Reply:
It would be easy enough to implement, changing `data TypeOrKind = Type Tree | Kind Tree` to `data TypeOrKind = Type Tree | Kind Tree | SortKind` (naming?). But I don't know how we would want to communicate this last case in our `primer-app` UI.

@dhess Any thoughts?

Reply:
I think I understand the question, but let's make sure. Is this about what type (aka "kind") to display when a kind is, say, `* -> * -> *`, and the student selects a non-root node in that tree?

Reply:
Prior to this PR, we have a function `Selection -> (Type | Kind)`. When any value-level node is selected it returns its type, and when any type-level node is selected we return its kind. The question is essentially: now that a selection can be of a kind-level node, what should be returned in that case?

(which I think is almost what you were saying, except that it includes root nodes)

Reply:
Let's discuss in the upcoming developer meeting.